src/Error/Error.php

```diff
@@ -172,7 +173,7 @@ public static function createLocatedError($error, $nodes = null, $path = null)
         }
 
         return new static(
-            $message ?: 'An unknown error occurred.',
+            $message !== '' ? $message : 'An unknown error occurred.',
```

Comment:
Can we invert the condition to avoid negation?

Reply:
393c555